sonaranalyzer-dotnet/its/expected/Ember-MM/Ember.Plugins-{9496C697-5AFD-4813-AEDC-AF33FACEADF0}-S101.json

```diff
@@ -2,20 +2,7 @@
 "issues":  [
 {
 "id":  "S101",
-"message":  "Rename class 'ShowFormOnUIThreadEventArgs' to match camel case naming rules, consider using 'ShowFormOnUiThreadEventArgs'.",
-"location":  {
-"uri":  "Ember-MM\Ember.Plugins\Events\ShowFormOnUIThread.cs",
-"region":  {
-"startLine":  17,
-"startColumn":  18,
-"endLine":  17,
-"endColumn":  45
-}
-}
-},
-{
-"id":  "S101",
-"message":  "Rename interface 'ITVImageScraper' to match camel case naming rules, consider using 'ITvImageScraper'.",
+"message":  "Rename interface 'ITVImageScraper' to match camel case naming rules, consider using 'ItvImageScraper'.",
```

Comment:
For me we should not raise at all here:
`I` for interface, `TV`, `Image`, `Scraper`
We might want to detect whether this is an interface to allow/prevent this. What I mean is, in this case, it's ok if this is an interface, it's not otherwise.